apps/dash-datashader/assets/style.css

```diff
@@ -30,7 +30,38 @@ h3 {
     justify-content: space-evenly;
 }
 
-@media only screen and (max-width: 800px) {
```

Comment:
Why are there changes to the stylesheet?

Reply:
Yunke said he had something to fix in the mobile layout